root/MaptickAverager.py

```diff
@@ -250,6 +283,35 @@ def add_new_value(list, new_value, value_creator):
         maptick_total += float(row["maptick"])
         player_total += float(row["players"])
         td_total += float(row["tidi_avg"]) 
+
+        sendmaps_total_cost = float(row.get("send_maps") or 0)#the last row read is the final value for these
+        sendmaps_total_calls = float(row.get("send_maps_count") or 0)#have to use .get(index) or 0 because not all logs have these indices
+        if sendmaps_total_cost != 0.0:
+            has_sendmaps_profiler_logs = True
+
+        sendmaps_client_cost = float(row.get("per_client") or 0)
+        sendmaps_client_calls = float(row.get("per_client_count") or 0)
+
+        movable_changes_cost = float(row.get("look_for_movable_changes") or 0)
+        movable_changes_calls = float(row.get("look_for_movable_changes_count") or 0)
+
+        check_turf_vis_contents_cost = float(row.get("check_turf_vis_conts") or 0)
+        check_turf_vis_contents_calls = float(row.get("check_turf_vis_conts_count") or 0)
+
+        check_hud_image_vis_contents_cost = float(row.get("check_hud/image_vis_contents") or 0)
+        check_hud_image_vis_contents_calls = float(row.get("check_hud/image_vis_contents_count") or 0)
+
+        loop_turfs_in_range_cost = float(row.get("turfs_in_range") or 0)
+        loop_turfs_in_range_calls = float(row.get("turfs_in_range_count") or 0)
+
+        obj_changes_cost = float(row.get("obj_changes") or 0)
+        obj_changes_calls = float(row.get("obj_changes_count") or 0)
+
+        mob_changes_cost = float(row.get("mob_changes") or 0)
+        mob_changes_calls = float(row.get("mob_changes_count") or 0)
+
+        movables_examined = float(row.get("movables_examined") or 0)
```

Comment:
Why set these vars once for each row when we only really care about the last one?

Reply:
done'd